src/oci.h

```diff
@@ -137,6 +137,11 @@
 /* Path to the stateless passwd file. */ 
 #define STATELESS_PASSWD_PATH "/usr/share/defaults/etc/passwd"
 
+/* Offset to add to the interface index for assigning the pci slot.
+ * First 3 slots are in use for pc-lite machine type
+ */
+#define PCI_OFFSET 4
```

Comment:
Sounds reasonable to me. @anthonyzxu - do you have any comments on this strategy?

@amshinde - how about adding a `g_debug()` call where the value of `PCI_OFFSET` is logged (in `cc_oci_expand_net_device_cmdline()`?), so that if there is a future clash with qemu-lite, atleast there is a log entry to aid in debugging.